src/lines.jl

```diff
@@ -9,12 +9,12 @@ function graphline(g, locs_x, locs_y, nodesize::Vector{T}, arrowlength, angleoff
         j = dst(e)
         Δx = locs_x[j] - locs_x[i]
         Δy = locs_y[j] - locs_y[i]
-        d  = sqrt(Δx^2 + Δy^2)
-        θ  = atan(Δy,Δx)
+        d = sqrt(Δx^2 + Δy^2)
```

Comment:
Maybe we should consider using `hypot(Δx, Δy)` here? It is slower though, but avoids some overflow issues.

Reply:
Let's go with the safer `hypot(Δx, Δy)` function.